explorer/app/components/Table/table.tsx

```diff
@@ -164,8 +181,8 @@ export const Table = <T extends object>({
                   {row.cells.map((cell, index) => {
                     return (
                       <TableCell {...cell.getCellProps()} key={index}>
-                        {cell.render("Cell")}
-                      </TableCell>
+                        {isLoading ? "Loading..." : cell.render("Cell")}
```

Comment:
Would you be able to move the `isLoading` check higher and also just show `<></>` for now if loading? Our mock for the loading state is currently table-wide (see [mocks](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/explore-dashboard?node-id=358%3A7778)).

Reply:
Keep its companion `// TODO` close to where you move the `isLoading` check!

Reply:
From the mocks, I noticed that we'll still render the table when loading, so I'm not returning `<></>` when loading. I'm presenting a `<span>Loading...</span>` instead.
I'm assuming we'll have a container div with `position: relative` and the loading component as a table sibling with `position: absolute`.
Is that ok?